package.json

```diff
@@ -155,7 +155,7 @@
         "fs-extra": "^10.0.0",
         "geckodriver": "4.0.3",
         "gh-pages": "^4.0.0",
-        "glob": "^10.3.0",
+        "glob": "^8.0.0",
```

Comment:
had to downgrade it, the latest one has CJS and MJS and the glob-promise is using the wrong one, didn't investigate further